docs/user-guide/hardware-setup.md

```diff
@@ -68,15 +71,15 @@ You will need Wi-Fi to communicate with your MAV when it is in the air.  ROS com
 
 For RC Control, you will need a transmitter with between 6 and 8 channels.  Any additional channels will be wasted.  We require RC control for safe operation, and only support arming and disarming via RC control.
 
-As of version 1.0, ROSflight only supports PPM (pulse position modulation) receivers. Support for serial RC (Spektrum Sattelites) is expected in future releases. A recommended RC setup is described below, but is meant as an example. Any configurations with PPM and 6-8 channels will be sufficient.
+As of version 1.0, ROSflight only supports PPM (pulse position modulation) receivers. A recommended RC setup is described below, but is meant as an example. Any configurations with PPM and 6-8 channels will be sufficient.
```

Comment:
Is SBUS supported?

Reply:
Not yet, I didn't feel like doing the plumbing for SBUS on this merge